data/actions/logement.publicodes

```diff
@@ -124,7 +124,7 @@ logement . éteindre appareils:
     Source : ADEME - [Réduire sa facture d'électricité](https://www.ademe.fr/sites/default/files/assets/documents/guide-pratique-reduire-facture-electricite.pdf), page 18.
 
 logement . éteindre appareils . empreinte du foyer:
-  formule: énergie de veille * commun . intensité électricité
+  formule: énergie de veille * électricité . intensité carbone
```

Comment:
En fait, je ne comprends pas la nécessité de `électricité . intensité carbone` et pourquoi pas utiliser directement `commun . intensité électricité` ?

Reply:
Car on pourrait avoir `logement . électricité . intensité carbone` différent de `commun . intensité électricité`

Reply:
Exemple si on considère le mix dans du logement variable selon la période, in fine `logement . électricité . intensité carbone` mais pas le mix de commun qui reste celui moyenb